src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ConnectionControl.xaml.cs

```diff
@@ -87,8 +87,8 @@ public bool UpdateConfigFromSelections(ConfigurationModel configuration)
                 if (issueConfigurationControl.CanSave)
                 {
                     string serializedConfigs = configuration.IssueReporterSerializedConfigs;
-                    Dictionary<Guid, string> configs = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigs);
-                    configs = configs ?? new Dictionary<Guid, string>();
+                    Dictionary<Guid, string> configs = string.IsNullOrEmpty(serializedConfigs) ? new Dictionary<Guid, string>()
+                        : JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigs);
```

Comment:
>DeserializeObject [](start = 38, length = 17)

This will throw if the data is somehow in a bad format. Where would that Exception be handled?